examples/BUILD.bazel

```diff
@@ -293,4 +329,7 @@ example_integration_test(
         "//packages/typescript:npm_package": "@bazel/typescript",
     },
     owners = ["@mattem"],
+    resolutions = {
```

Comment:
maybe we should set this as the default value for the `resolutions` attribute so we don't have to remember where it has to be set? wouldn't do any harm to include it even for tests that don't require it, right?

Reply:
the only problem I see is having to build this package but bazel is smart enough to use from previous builds so that makes sense to put this as a default. will send a pr for it.